packages/graphiql/src/locales/ru/Errors.json

```diff
@@ -0,0 +1,7 @@
+{
+  "GraphiQL requires a fetcher function": "GraphiQL запрашивает функцию выборки",
```

Comment:
and we can just remove this line, or make the righthand value the english for the new error until we get a new translation for this locale string